src/main/java/com/denizenscript/clientizen/objects/EntityTag.java

```diff
@@ -179,6 +186,18 @@ public static void register() {
             }
             return null;
         });
+
+        // <--[tag]
+        // @attribute <EntityTag.visible_to_camera>
+        // @returns ElementTag
+        // @group Clientizen
+        // @description
+        // Returns whether an entity is visible to any client's camera.
```

Comment:
`any client's` there will only ever be one client at a time (the one Clientizen runs in), unless this somehow returns whether the entity is visible to any of the online player's clients?

Reply:
This was a long time ago, so I can't remember so I'll retest.